common/changes/@microsoft.azure/openapi-validator-rulesets/tejaswis-test-XmsPageableForListCalls_2024-06-19-19-02.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
```

Comment:
Note that you no longer need to include these changelog json files:

https://github.com/Azure/azure-openapi-validator/issues/683